src/schinkenfleckerl-gratinated.md

```diff
@@ -19,21 +23,19 @@ Serve topped with cheese on top of salad (with Mustard, Jam, Salt, 1 Vinegar, 4
 ## Directions
 
 1. Start cooking noodles.
-2. Cube ham into smaller than 1cm³ pieces.
-  - slice
-  - make 3 slices at once into cubes
+2. First slice, then cube ham into smaller than 1cm³ pieces.
 3. Whip cream in a big bowl.
 4. Add spices, eggs and sourcream to bowl and stir.
 5. Put ham cubes in and stir.
 6. Put noodles without water into bowl and mix.
-7. Put the mixture into one or two baking pans (with baking paper)
-8. Bake at ~160°C for ~2 hours
+7. Put the mixture into one or two baking pans (with baking paper).
+8. Bake at ~160°C for ~2 hours.
 9. Optionally gratin Mozarella or any other cheese on top for 5-10 minutes.
 
 There you have it, an idiot-proof carb and protein bomber that should taste reasonably well. Besides browning the noodles this is really just throwing stuff into a baking pan. This is the kind of dish that tastes better the next day so it's ideal for making on a Sunday and eating for the next 3 or 4 days. Just reheat in oven or pan (add little water).
 
 ## Contribution
 
-- Martin Stohanzl - [website](loesmartin.at), [youtube](https://www.youtube.com/channel/UCJkTyM5-fcVBkhi7DypoNsg)
+- Martin Stohanzl - [website](https://loesmartin.at), [youtube](https://www.youtube.com/channel/UCJkTyM5-fcVBkhi7DypoNsg)
 
 ;tags: austrian beef pork ham noodles smoked-rolled-roast baking-pan
```

Comment:
7 tags are too much. Limit these to 3-4 at most.

Reply:
2-3 is probably a better limit for tags